artwork-data/artworks-overrides.json

```diff
@@ -2576,7 +2576,7 @@
       "country": "United States",
       "postalCode": "95113",
       "state": "CA",
-      "sourceURL": "https://www.powwowsanjose.com/"
+      "sourceURL": "https://www.sanjoseca.gov/home/showdocument?id=32425"
```

Comment:
Change this link to https://www.powwowsanjose.com/mural-locations

Reply:
Am I to understand that all dead powwowsanjose links should be changed to:
https://www.powwowsanjose.com/mural-locations
If so I can do that.

Reply:
Please do, thank you!

Reply:
OK, that should be done in my latest push.